Models/MuZero_torch_agent.py

```diff
@@ -200,14 +248,14 @@ def __init__(self,
             max_value = contractive_mapping(max_value)
             min_value = contractive_mapping(min_value)
         if num_steps <= 0:
-            num_steps = torch.ceil(max_value) + 1 - torch.floor(min_value)
+            num_steps = np.ceil(max_value) + 1 - np.floor(min_value)
         self.min_value = min_value
         self.max_value = max_value
         self.value_range = max_value - min_value
```

Comment:
Check if we are using this class at all. If not, we want to delete this extra code.